src/coreclr/vm/codeman.cpp

```diff
@@ -4294,6 +4294,7 @@ BOOL EECodeGenManager::JitCodeToMethodInfoWorker(
 #ifdef FEATURE_EH_FUNCLETS
         // Computed lazily by code:EEJitManager::LazyGetFunctionEntry
         pCodeInfo->m_pFunctionEntry = NULL;
+        pCodeInfo->m_isFuncletCache = pCHdr->ComputeInitialIsFuncletCacheValue(currentPC);
```

Comment:
```suggestion
        if (pCHdr->MayHaveFunclets())
        {
            // Computed lazily by code:EEJitManager::LazyGetFunctionEntry
            pCodeInfo->m_pFunctionEntry = NULL;
            pCodeInfo->m_isFuncletCache = IsFuncletCache::NotSet);
        }
        else
        {
            pCodeInfo->m_pFunctionEntry = pCHdr->GetUnwindInfo(0)(;
            pCodeInfo->m_isFuncletCache = IsFuncletCache::IsNotFunclet;
        }
```
We may be able to do even more cheap caching for the common case here.

Reply:
Nit: Typo - s/`(0)(`/`(0)`.

I already had this optimization locally and it's useful.